lib/interfaces/src/MCP23017Interface.cpp

```diff
@@ -0,0 +1,13 @@
+#include "MCP23017Interface.h"
+
+#include "SharedFirmwareTypes.h"
+
+namespace specifiedBit
+{
+    bool getBit(uint16_t data, bool port, int bit){
```

Comment:
to provide implementation, I'd just use `bool specifiedBit::getBit(uint16_t data, ...`

Reply:
instead of another namespace declaration